indico/modules/rb_new/controllers/user/landing.py

```diff
@@ -16,10 +16,52 @@
 
 from __future__ import unicode_literals
 
+from datetime import datetime, time
+
+import dateutil
+from flask import request, session
+
+from indico.modules.rb import rb_settings
 from indico.modules.rb.controllers import RHRoomBookingBase
+from indico.modules.rb.models.rooms import Room
 from indico.modules.rb_new.views.base import WPRoomBookingBase
+from indico.util.string import natural_sort_key
+from indico.web.util import jsonify
 
 
 class RHRoomBookingLanding(RHRoomBookingBase):
     def _process(self):
         return WPRoomBookingBase.display('landing.html')
+
+
+class RHNewRoomBookingNewBooking(RHRoomBookingBase):
```

Comment:
1. In my opinion, there's no need to call it `NewRoomBookingNewBooking`, since anyway we're already in the `rb_new` namespace
2. This is not supposed to be a "new booking" handler anymore, but rather a "room search" or "room booking search".

Reply:
Agreed. Forgot to rename it after moving it to the rooms_new module